### PR TITLE
ID Localization for Install Kubectl

### PR DESCRIPTION
**Pull request by handywijaya**
- Add bahasa translation for /docs/tasks/tools/install-kubectl

Issue: https://github.com/kubernetes/website/issues/20111

### CODE REVIEW

**Comment by handywijaya** [comment]
Hi @k8s-ci-robot I have just registered a CLA, please run the check again :laughing:

**Comment by handywijaya** [comment]
forced update to change commit author

**Comment by handywijaya** [comment]
another forced update to change committer. sorry a bit mess here :sweat_smile:

**Comment by handywijaya** [comment]
sorry, I have just completed the CLA registration process.

File: content/id/docs/tasks/tools/install-kubectl.md [hunk 1]
```diff
@@ -0,0 +1,496 @@
...
+content_template: templates/task
+weight: 10
+card:
+  name: tasks
+  weight: 20
+  title: Instalasi kubectl
+---
+
+{{% capture overview %}}
+[Kubectl](/docs/user-guide/kubectl/) adalah perangkat _command line_ Kubernetes yang digunakan untuk menjalankan perintah-perintah untuk kluster Kubernetes. Kamu dapat menggunakan `kubectl` untuk men-_deploy_ aplikasi, mengatur _resource_ kluster, dan melihat _log_. Daftar operasi `kubectl` dapat dilihat di [Ikhtisar kubectl](/docs/reference/kubectl/overview/).
```
Comment (girikuncoro) [inline content/id/docs/tasks/tools/install-kubectl.md:12]: how about translating command line to "barisan perintah"?

File: content/id/docs/tasks/tools/install-kubectl.md [hunk 2]
```diff
@@ -0,0 +1,496 @@
...
+
+    Untuk mengunduh versi spesifik, ganti bagian `curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt` dengan versi yang diinginkan.
+
+    Misal, untuk mengunduh versi {{< param "fullversion" >}} di Linux, ketik:
+    
+    ```
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/{{< param "fullversion" >}}/bin/linux/amd64/kubectl
+    ```
+
+1. Buat agar binari `kubectl` dapat di-_execute_.
```
Comment (girikuncoro) [inline content/id/docs/tasks/tools/install-kubectl.md:39]: execute = eksekusi or jalankan
```suggestion
1. Buat agar binari `kubectl` dapat dijalankan.
```

File: content/id/docs/tasks/tools/install-kubectl.md [hunk 3]
```diff
@@ -0,0 +1,496 @@
...
+
+    Untuk mengunduh versi spesifik, ganti bagian `curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt` dengan versi yang diinginkan.
+
+    Misal, untuk mengunduh versi {{< param "fullversion" >}} di macOS, ketik:
+		  
+    ```
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/{{< param "fullversion" >}}/bin/darwin/amd64/kubectl
+    ```
+
+1. Buat agar binari `kubectl` dapat di-_execute_.
```
Comment (girikuncoro) [inline content/id/docs/tasks/tools/install-kubectl.md:119]:
```suggestion
1. Buat agar binari `kubectl` dapat dijalankan.
```

File: content/id/docs/tasks/tools/install-kubectl.md [hunk 4]
```diff
@@ -0,0 +1,496 @@
...
+
+    ```
+    kubectl version --client
+    ```
+
+    {{< note >}}Proses pembaruan dapat dilakukan dengan menjalankan ulang dua perintah yang terdapat pada langkah 1.{{< /note >}}
+
+### Instalasi di Windows menggunaakn Chocolatey atau Scoop
+
+Untuk menginstal `kubectl` di Windows kamu dapat menggunakan paket manajer [Chocolatey](https://chocolatey.org) atau _installer_ command-line [Scoop](https://scoop.sh).
```
Comment (girikuncoro) [inline content/id/docs/tasks/tools/install-kubectl.md:223]:
```suggestion
Untuk menginstal `kubectl` di Windows kamu dapat menggunakan paket manajer [Chocolatey](https://chocolatey.org) atau _installer_ perintah barisan [Scoop](https://scoop.sh).
```

File: content/id/docs/tasks/tools/install-kubectl.md [hunk 5]
```diff
@@ -0,0 +1,496 @@
...
+    
+1. Pastikan instalasi sudah berhasil dengan melakukan pengecekan versi:
+
+    ```
+    kubectl version --client
+    ```
+
+## Memeriksa konfigurasi kubectl
+
+Agar `kubectl` dapat mengakses kluster Kubernetes, dibutuhkan sebuah [kubeconfig file](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/), yang akan otomatis dibuat ketika kamu membuat kluster baru menggunakan [kube-up.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-up.sh) atau setelah berhasil men-_deploy_ kluster Minikube. Secara _default_, konfigurasi `kubectl` disimpan di `~/.kube/config`.
```
Comment (girikuncoro) [inline content/id/docs/tasks/tools/install-kubectl.md:286]: file = berkas in Bahasa Indonesia
```suggestion
Agar `kubectl` dapat mengakses kluster Kubernetes, dibutuhkan sebuah [berkas kubeconfig](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/), yang akan otomatis dibuat ketika kamu membuat kluster baru menggunakan [kube-up.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-up.sh) atau setelah berhasil men-_deploy_ kluster Minikube. Secara _default_, konfigurasi `kubectl` disimpan di `~/.kube/config`.
```

File: content/id/docs/tasks/tools/install-kubectl.md [hunk 6]
```diff
@@ -0,0 +1,496 @@
...
+
+Tetapi jika kamu melihat pesan seperti di bawah maka `kubectl` belum dikonfigurasi dengan benar atau tidak dapat terhubung ke kluster Kubernetes.
+
+```shell
+The connection to the server <server-name:port> was refused - did you specify the right host or port?
+```
+
+Selanjutnya, apabila kamu ingin menjalankan kluster Kubernetes di laptop (lokal), kamu memerlukan sebuah perangkat seperti Minikube sebelum menjalankan ulang perintah yang ada di atas.
+
+Jika `kubectl cluster-info` mengembalikan response URL tetapi kamu masih belum dapat mengakses ke kluster, kamu bisa menggunakan perintah di bawah untuk memeriksa apakah kluster sudah dikonfigurasi dengan benar.
```
Comment (girikuncoro) [inline content/id/docs/tasks/tools/install-kubectl.md:303]: response = "kembalian" or "respons" (https://kbbi.web.id/respons)
```suggestion
Jika `kubectl cluster-info` mengembalikan respons URL tetapi kamu masih belum dapat mengakses ke kluster, kamu bisa menggunakan perintah di bawah untuk memeriksa apakah kluster sudah dikonfigurasi dengan benar.
```

File: content/id/docs/tasks/tools/install-kubectl.md [hunk 7]
```diff
@@ -0,0 +1,496 @@
...
+
+_Completion script_ `kubectl` untuk Bash dapat dibuat dengan perintah `kubectl completion bash`. Masukkan skrip tersebut ke dalam terminal sebagai sumber untuk menyalakan _auto complete_ dari `kubectl`.
+
+Namun, _completion script_ tersebut tergantung dengan [**bash-completion**](https://github.com/scop/bash-completion), yang artinya kamu harus menginstal program tersebut terlebih dahulu (kamu dapat memeriksa apakah kamu sudah memiliki bash-completion dengan menjalankan perintah `type _init_completion`).
+
+### Instalasi bash-completion
+
+bash-completion disediakan oleh banyak manajer paket (lihat [di sini](https://github.com/scop/bash-completion#installation)). Kamu dapat menginstalnya dengan menggunakan perintah `apt-get install bash-completion` atau `yum install bash-completion`, atau dsb.
+
+Perintah di atas akan membuat skrip utama bash-completion di `/usr/share/bash-completion/bash_completion`. Terkadang kamu juga harus menambahkan skrip tersebut ke dalam _file_ `~/.bashrc`, tergantung paket manajer yang kamu pakai.
```
Comment (girikuncoro) [inline content/id/docs/tasks/tools/install-kubectl.md:331]: file = berkas, same comment for the rest of this page
```suggestion
Perintah di atas akan membuat skrip utama bash-completion di `/usr/share/bash-completion/bash_completion`. Terkadang kamu juga harus menambahkan skrip tersebut ke dalam berkas `~/.bashrc`, tergantung paket manajer yang kamu pakai.
```

File: content/id/docs/tasks/tools/install-kubectl.md [hunk 8]
```diff
@@ -0,0 +1,496 @@
...
+
+### Pembaruan Bash
+
+Panduan di bawah berasumsi kamu menggunakan Bash 4.1+. Kamu dapat memeriksa versi Bash dengan menjalankan:
+
+```shell
+echo $BASH_VERSION
+```
+
+Jika versinya sudah terlalu usang, kamu dapat menginstal/membaruinya dengan menggunakan Homebrew:
```
Comment (girikuncoro) [inline content/id/docs/tasks/tools/install-kubectl.md:394]: nit: memperbarui
```suggestion
Jika versinya sudah terlalu usang, kamu dapat menginstal/memperbarui-nya dengan menggunakan Homebrew:
```

File: content/id/docs/tasks/tools/install-kubectl.md (hunk 1 above)
Comment (handywijaya) [reply content/id/docs/tasks/tools/install-kubectl.md:12]: I think "berbagai perintah" sounds better. wdyt @girikuncoro ?

File: content/id/docs/tasks/tools/install-kubectl.md (hunk 8 above)
Comment (handywijaya) [reply content/id/docs/tasks/tools/install-kubectl.md:394]: should we put `-`? how about just `memperbaruinya`? @girikuncoro

Comment (girikuncoro) [reply content/id/docs/tasks/tools/install-kubectl.md:394]: sure that works as well

File: content/id/docs/tasks/tools/install-kubectl.md (hunk 1 above)
Comment (girikuncoro) [reply content/id/docs/tasks/tools/install-kubectl.md:12]: I think barisan perintah (command-line) is also used in other translations: https://git-scm.com/book/ms/v2/Untuk-Bermula-Barisan-Perintah-ataupun-Command-Line

Comment (handywijaya) [reply content/id/docs/tasks/tools/install-kubectl.md:12]: Oh I'm sorry, I got the wrong highlight, I thought the one you mention was to replace "perintah-perintah", turns out the "_command line_" one.
Yeah, "barisan perintah" sounds good, I'll change the others too. Thanks @girikuncoro!

**Pull request merged**